src/patternfly/components/Popover/popover.scss

```diff
@@ -30,18 +28,6 @@
   // Header variables
   --pf-c-popover__header--MarginBottom: var(--pf-global--spacer--md);
 
-  // Header title variables
-  --pf-c-popover__header-title--FontSize: var(--pf-c-title--m-3xl--FontSize);
-  --pf-c-popover__header-title--FontWeight: var(--pf-c-title--m-3xl--FontWeight);
```

Comment:
From the [original issue](https://github.com/patternfly/patternfly-next/issues/184) it looks like this should be `.pf-c-title.pf-m-lg`? or maybe `.pf-m-xl`? You could verify with @mcarrano.